src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormArtistStep.tsx

```diff
@@ -27,9 +62,70 @@ export const MyCollectionArtworkFormArtistStep: React.FC<MyCollectionArtworkForm
 
       <Spacer y={4} />
 
-      {/* TODO: Implement screen */}
+      <Text variant={"lg"}>Select an Artist</Text>
+
+      <Spacer y={4} />
+
+      <ArtistAutoComplete
+        onError={onError}
+        onChange={value => {
+          setQuery(value)
+          console.log(value)
+        }}
+        onSelect={onSelect}
+        placeholder="Search for artists on Artsy"
+        title={""}
+      />
+
+      <Spacer y={2} />
+
+      <Flex flexDirection="row">
+        <Text variant="xs">Can't find the artist? &nbsp;</Text>
+        <Clickable
+          onClick={() => onSkipPress?.(trimmedQuery)}
+          textDecoration="underline"
+        >
+          <Text variant="xs" color="black100">
+            Add their name.
+          </Text>
+        </Clickable>
+      </Flex>
+
+      <Spacer y={4} />
+
+      <Text variant="xs">Artists in My Collection</Text>
+
+      <Spacer y={3} />
 
-      <Text variant={"xl"}>Select Artist</Text>
+      {collectedArtists.map((artist, index) => {
+        return (
+          <Flex
+            key={artist.internalID}
+            flexDirection="row"
+            justifyContent="space-between"
+            alignItems="center"
+            mt={1}
+          >
+            <Clickable
+              onClick={() =>
+                onSelect({
+                  artistId: artist.internalID,
+                  artistName: artist.displayLabel,
+                })
+              }
+            >
+              <Avatar
+                size="xs"
+                src="https://picsum.photos/45/45"
+                srcSet="https://picsum.photos/45/45 1x, https://picsum.photos/90/90 2x"
+                lazyLoad
+                initials="TK"
```

Comment:
👀

Reply:
It's now implemented. What you saw here was just a placeholder 👍

Reply:
awesome